### PR TITLE
chore: 🤖 enable ap-northeast-3 region

### PR DESCRIPTION
**Pull request by petercwk-aws**
#### Description of changes
This PR enables support in ap-northeast-3 region (KIX)

#### Issue #, if available
Associated amplify-category-api PR: https://github.com/aws-amplify/amplify-category-api/pull/1741

#### Description of how you validated changes
I set ap-northeast-3 in my ~/.aws/config for my amplify profile and created an app.

I ran: 
amplify-dev init
amplify-dev add api (default graphQL options)
amplify-dev push

Confirmed that AppSync api was deployed to my account in the ap-northeast-3 region.

Ran e2e tests here: https://us-east-1.console.aws.amazon.com/codesuite/codebuild/671107461633/projects/AmplifyCLI-E2E-Testing/batch/AmplifyCLI-E2E-Testing:61be36be-ad87-40be-b52e-471989855ee6?region=us-east-1#

#### Checklist

- [X] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const Translate = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': 'ap-northeast-1',
   'ap-northeast-2': 'ap-northeast-2',
+  'ap-northeast-3': 'ap-northeast-1',
```
Comment (petercwk-aws) [inline packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js:22]: Translate is not available in ap-northeast-3. Falling back to ap-northeast-3

https://docs.aws.amazon.com/general/latest/gr/translate-service.html

File: packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js [hunk 2]
```diff
@@ -61,6 +63,7 @@ const Transcribe = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': latestTranscribeRegions >= 1 ? 'ap-northeast-1' : 'ap-northeast-2',
   'ap-northeast-2': 'ap-northeast-2',
+  'ap-northeast-3': 'ap-northeast-1',
```
Comment (petercwk-aws) [inline packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js:66]: Transcribe is not available in ap-northeast-3. Falling back to ap-northeast-1.

https://docs.aws.amazon.com/general/latest/gr/transcribe.html

File: packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js [hunk 3]
```diff
@@ -82,6 +85,7 @@ const Rekognition = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': 'ap-northeast-1',
   'ap-northeast-2': 'ap-northeast-2',
+  'ap-northeast-3': 'ap-northeast-1',
```
Comment (petercwk-aws) [inline packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js:88]: Rekognition is not available in ap-northeast-3. Falling back to ap-northeast-1.

https://docs.aws.amazon.com/general/latest/gr/rekognition.html

File: packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js [hunk 4]
```diff
@@ -103,6 +107,7 @@ const RekognitionAndTextract = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': 'us-east-1',
   'ap-northeast-2': latestTextractRegions >= 1 ? 'ap-northeast-2' : 'us-east-1',
+  'ap-northeast-3': 'ap-northeast-2',
```
Comment (petercwk-aws) [inline packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js:110]: Textract is not available in ap-northeast-3. Falling back to ap-northeast-2.

https://docs.aws.amazon.com/general/latest/gr/textract.html

File: packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js [hunk 5]
```diff
@@ -124,6 +129,7 @@ const Comprehend = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': latestComprehendRegions >= 1 ? 'ap-northeast-1' : 'us-east-1',
   'ap-northeast-2': latestComprehendRegions >= 1 ? 'ap-northeast-2' : 'us-east-1',
+  'ap-northeast-3': 'ap-northeast-1',
```
Comment (petercwk-aws) [inline packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js:132]: Comprehend is not available in ap-northeast-3. Falling back to ap-northeast-1.

https://docs.aws.amazon.com/general/latest/gr/comprehend.html

File: packages/amplify-provider-awscloudformation/src/aws-regions.js [hunk 6]
```diff
@@ -14,6 +14,7 @@ const regionMappings = {
   'eu-central-1': 'EU (Frankfurt)',
   'ap-northeast-1': 'Asia Pacific (Tokyo)',
   'ap-northeast-2': 'Asia Pacific (Seoul)',
+  'ap-northeast-3': 'Asia Pacific (Osaka)',
```
Comment (sobolk) [inline packages/amplify-provider-awscloudformation/src/aws-regions.js:17]: please read and follow a note from the comment that's at the beginning of this file.

**Review by sobolk — APPROVED** [review]
Good to merge with e2e tests passing.
Can you run `yarn cloud-e2e` or `yarn cloud-e2e-circleci` ?

File: packages/amplify-provider-awscloudformation/src/aws-regions.js (hunk 6 above)
Comment (petercwk-aws) [reply packages/amplify-provider-awscloudformation/src/aws-regions.js:17]: My apologies, last commit addresses this.

**Comment by johnpc** [comment]
I ran cloud e2e here: https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli?branch=run-e2e/xss/cli/kix

**Review by Amplifiyer — APPROVED** [review]

**Comment by petercwk-aws** [comment]
Rebased dev and ran e2e tests here: https://us-east-1.console.aws.amazon.com/codesuite/codebuild/671107461633/projects/AmplifyCLI-E2E-Testing/batch/AmplifyCLI-E2E-Testing:61be36be-ad87-40be-b52e-471989855ee6?region=us-east-1#

**Pull request merged**